targets/mega65/mega65.c

```diff
@@ -685,8 +697,17 @@ static void emulation_loop ( void )
 #ifdef		HAS_UARTMON_SUPPORT
 		if (XEMU_UNLIKELY(breakpoint_pc == cpu65.pc)) {
 			DEBUGPRINT("TRACE: Breakpoint @ $%04X hit, Xemu moves to trace mode after the execution of this opcode." NL, cpu65.pc);
+			m65mon_show_regs();
 			paused = 1;
 		}
+		if (watchpoint_addr != -1)
```

Comment:
I recall from past PRs, you were sensitive to adding extra logic into the main loop, understood.

This is just the first implementation approach that came to mind. If you feel this isn't a good way to go, no problem. If you can think of another/better way, ok, let's try that. If not, no worries, let's skip 'w' and save it for another time.